packages/framework/src/Form/Constraints/MoneyRangeValidator.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Form\Constraints;
+
+use Shopsys\FrameworkBundle\Component\Money\Money;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class MoneyRangeValidator extends ConstraintValidator
+{
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Money\Money|null $value
+     * @param \Symfony\Component\Validator\Constraint $constraint
+     */
+    public function validate($value, Constraint $constraint)
+    {
+        if (!$constraint instanceof MoneyRange) {
+            throw new \Symfony\Component\Validator\Exception\UnexpectedTypeException($constraint, MoneyRange::class);
+        }
+
+        if ($value instanceof Money) {
+            if ($constraint->max !== null && $value->isGreaterThan($constraint->max)) {
+                $this->context->addViolation($constraint->maxMessage, [
+                    '{{ limit }}' => $constraint->max->toString(),
+                ]);
+            }
+
+            if ($constraint->min !== null && $value->isLessThan($constraint->min)) {
+                $this->context->addViolation($constraint->minMessage, [
+                    '{{ limit }}' => $constraint->min->toString(),
+                ]);
+            }
+        } elseif ($value !== null) {
```

Comment:
Please rearrange IF - as same as with `NotNegative...` - move `if ($value === null) {return;}` above chechking `Money`, and then throwing an exception can be without if again